README.md

```diff
@@ -139,4 +139,8 @@ NAME                     STATUS   ROLES   AGE   VERSION
 aks-default-75135322-0   Ready    agent   18m   v1.13.12
 aks-default-75135322-1   Ready    agent   18m   v1.13.12
 aks-default-75135322-2   Ready    agent   18m   v1.13.12
-```
\ No newline at end of file
+```
+Kubernetes Dashboard 
```

Comment:
I would document the whole process of accessing the Kubernetes dashboard that is described [here](https://github.com/anubhavmishra/terraform-azurerm-aks/issues/3#issuecomment-551076860).